docker/install.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 set -e
 
-# Checkout dependent repositories
-/home/carma/.base-image/ros1_msgs_ws/src/carma_msgs/docker/checkout.bash -r /home/carma/.base-image/workspace
+# Receiving the GIT_BRANCH argument and echoing it for debugging
+echo "Arguments received:"
+echo "GIT_BRANCH: $1"
+GIT_BRANCH=$1 # Assigning the first argument to GIT_BRANCH and can be used in script below.
```

Comment:
I think we can remove these new lines since the `GIT_BRANCH` argument is not used in this install.sh script.

Reply:
same as above comment I thought we can retrieve and use them further cases. I can remove these